graphene_django/rest_framework/mutation.py

```diff
@@ -67,9 +90,35 @@ def __init_subclass_with_meta__(cls, serializer_class=None,
         )
         super(SerializerMutation, cls).__init_subclass_with_meta__(_meta=_meta, input_fields=input_fields, **options)
 
+    @classmethod
+    def resolve_serializer_inputs(cls, root, info, **input):
```

Comment:
What do you think of `get_serializer_kwargs` for this method? "resolve" and "inputs" both have special meanings in the graphene world.

Reply:
I like it 👍